integration_tests/test_intrinsic_function_mixed_print.py

```diff
@@ -0,0 +1,17 @@
+from lpython import i32
+
+def test_intrinsic_function_mixed_print():
+    # list and list methods
+    my_list: list[i32] = [1, 2, 3, 4, 5]
+    print("Popped element:", my_list.pop())
+    print("1 is located at:", my_list.index(1))
+    my_list.append(2)
+    print("2 is present", my_list.count(2), "times")
+    print(my_list.pop(), my_list)
+
+    # dict and dict methods
+    my_dict: dict[str, i32] = {"first": 1, "second": 2, "third": 3}
+    print("Keys:", my_dict.keys())
+    print("Value of 'third':", my_dict.pop("third"))
+
```

Comment:
Can you also add asserts in this test and also enable it in integration_tests? Its good to have this in both places integration_tests and reference runtime output.

Reply:
Do you mind adding appropriate asserts wherever possible in the test?

Reply:
@Shaikh-Ubaid what would we assert? Aren't we only printing here?

Reply:
Ok. You can assert on the value that got popped and the list that is left after popping.

Reply:
Done.